src/Hydration.js

```diff
@@ -1,25 +1,58 @@
 class Hydration {
-  constructor(hydrationData) {
-    this.hydrationData = hydrationData;
+  constructor(hydrationData, user) {
+    this.userId = user.id;
+    this.hydrationData = this.getHydrationData(hydrationData);
   }
-
-  calculateAverageOunces(id) {
-    const perDayUserHydration = this.hydrationData.filter((data) => id === data.userID);
-    return perDayUserHydration.reduce((sumSoFar, data) => sumSoFar += data.numOunces, 0) / perDayUserHydration.length;
+  
+  getHydrationData(hydrationData) {
```

Comment:
We probably need to leave an ID's related to the raw data to capital ID, even though we agreed in our project we would use Id. Hopefully this makes sense!

Reply:
Got it will change that to an upper ID